src/js/wand.js

```diff
@@ -1,24 +1,31 @@
-(function() {
-  Wand = function(options) {
+var Wand = (function() {
+
+  var wand = {};
+
+  var options = {},
+      nodes = [],
+      elem = null;
+
+  var badHtmlError = new Error('You did not pass a valid HTML Element');
+
+  wand.init = function(opts) {
```

Comment:
If possible, I'd like to be able to run the `init` method when you create a new `Wand`. I'm thinking of something like [`datatables`](http://www.datatables.net/) or [`bootstrap-datepicker`](https://bootstrap-datepicker.readthedocs.org/en/latest/) where you don't have to call an explicit initialization method, but just instead do something like 

```
$('#myDatepicker').datepicker()
```